README.md

```diff
@@ -27,6 +27,20 @@ These are the steps to add your own component:
 
 More examples of using Kustomize to drive deployments using GitOps can be [found here](https://github.com/redhat-cop/gitops-catalog).
 
+## Component testing
```

Comment:
Could we rephrase this - since these pipelines may do test and build ( images) as most of the pipelines today are doing ?

Reply:
Sure, just updated to "Component testing and building of images"